web/react/components/post_list.jsx

```diff
@@ -83,6 +83,7 @@ export default class PostList extends React.Component {
         };
     }
     componentDidMount() {
+        window.onload = () => this.scrollToBottom();
```

Comment:
The only issue I can see is strangeness if we where to implement a scroll position remembering feature. But if you have tested it, it's all good.